src/view/Dialog.tsx

```diff
@@ -0,0 +1,105 @@
+            <div className="dialog-top">
+                <div className="aktivitet">
+                    <Lenke href="{() => {console.log('skal følge lenke til altivitet');}">
+                    <Undertittel>
+                        Aktivitet: {aktivitet}
+                        <HoyreChevron stor className="aktivitet-chevron" type={'høyre'}/>
+                    </Undertittel>
+                </Lenke></div>
+                <div className="venter-pa">
+                    <div className="venter-pa-item">
+                        <Checkbox
+                            id='navHarBallen'
+                            label='Venter på svar fra NAV'
+                            checked={!dialogensdata.venterPaSvar}
+                            className="venter-pa-item-checkbox"
+                            onChange={() => alert('TODO: Skal check toggle "venterPåSvar"?')}
+                        />
+                    </div>
+                    <div className="venter-pa-item">
+                        <Checkbox
+                            id='brukerHarBallen'
+                            label='Venter på svar fra bruker'
+                            checked={dialogensdata.venterPaSvar}
+                            className="venter-pa-item-checkbox"
+                            onChange={() => alert('TODO: Skal check toggle "venterPåSvar"?')}
+                        />
+                    </div>
+                </div>
+                <hr/>
+                <div className="skriv-melding">
+                    <Textarea
+                        label='Skriv en melding til brukeren'
+                        textareaClass='meldingsfelt'
+                        id='meldingIn'
+                        name='NyMelding'
+                        onChange={(event) => setValue((event.target as HTMLInputElement).value)}
+                        value={value}
+                    /> &nbsp;
+                    <Hovedknapp title={"Send"}
+                                onClick={() => sendeNyMelding({tekst: value})}
+                    >Send</Hovedknapp>
+                </div>
+            </div>
+            <hr/>
+            <div className="dialog-pane dialog-bottom">
+                <HenvendelseList henvendelseData={dialogensdata.henvendelser}/>
+            </div>
+        </div>
+    }
+
+    if (props.dialogensdata !== null) {
+        aktivitet = (aktivitet !== null || aktivitet !== undefined ) ? "ingen" : props.dialogensdata.aktivitetId;
+        return (
+            showDialog(props.dialogensdata)
+        )
+    } else {
+        return (
+            <div className="dialog">
+                <div className="dialog-pane dialog-top">
+                    <hr/>
+                    <Innholdstittel> Ingen Valgt Dialog</Innholdstittel>
+                    <hr/>
+                </div>
+                <div className="dialog-pane dialog-bottom">
+                    <hr/>
+                    <HenvendelseList henvendelseData={[{
```

Comment:
Er ikke denne biten litt rart ? Hvis dialogen mangler, da mangler jo også henvendelser ?

Reply:
Fjernet logikken for å vise tom henvendelsesliste